test/support/index.js

```diff
@@ -29,3 +27,12 @@ exports.expectError = function OErrorExpectError(e, expected) {
   // first stack frame should be the function where the error was thrown
   expect(e.stack.split('\n')[1]).to.match(expected.firstFrameRx)
```

Comment:
Just a thought: these comments before the assertion could be passed as failure info message. A way to have self-documented code.

```
expect(e.stack.split('\n')[1]).to.match(expected.firstFrameRx
             'first stack frame should be the function where the error was thrown')
```

Reply:
Nice! Added in 131a5e4.